app/v2/models/errors/mtdErrors.scala

```diff
@@ -129,6 +130,13 @@ object UnauthorisedError extends MtdError("CLIENT_OR_AGENT_NOT_AUTHORISED", "The
 
 object InvalidBearerTokenError extends MtdError("UNAUTHORIZED", "Bearer token is missing or not authorized")
 
+//Stub errors
+object RuleIncorrectGovTestScenarioError
+    extends MtdError(
+      code = "RULE_INCORRECT_GOV_TEST_SCENARIO",
+      message = "The Gov-Test-Scenario was not found"
+    )
+
```

Comment:
Could we change the format to match the other errors?

Reply:
Yes we can